bin/pd/Makefile

```
@@ -145,7 +145,7 @@ ati: upload-ati rebuild-ati
 
 .PHONY: upload-ati
 upload-ati: $(workdir)/filtered/ati.csv $(workdir)/filtered/ati-nil.csv
-	@echo Uploading ATI Summaries...
+	@echo $(shell date): Uploading ATI Summaries...
```

How about something like
```suggestion
	@$(echo_date) Uploading ATI Summaries...
```
so that we can define `echo_date` above to include `echo` and whatever date format prefix we choose. That would let us use the One True Date format without repeating code all over the Makefile.